src/plugins/data_views/server/saved_objects_client_wrapper.ts

```diff
@@ -42,6 +42,6 @@ export class SavedObjectsClientServerToCommon implements SavedObjectsClientCommo
     return await this.savedObjectClient.create(type, attributes, options);
   }
   delete(type: string, id: string) {
-    return this.savedObjectClient.delete(type, id);
+    return this.savedObjectClient.delete(type, id, { force: true });
```

Comment:
whats the `force:true` for ?

Reply:
it is necessary to delete saved objects that are available in multiple spaces.